x-pack/osquerybeat/ext/osquery-extension/pkg/tables/elastic_file_analysis.go

```diff
@@ -0,0 +1,102 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package tables
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/elastic/beats/v7/x-pack/osquerybeat/internal/command"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+func FileAnalysisColumns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.TextColumn("path"),
+		table.TextColumn("mode"),
+		table.TextColumn("uid"),
```

Comment:
also maybe use 
```
table.BigIntColumn("uid"),
table.BigIntColumn("gid"),
```

for schema.

The osquerybeat converts results to the proper types columns based on the known column type (schema), so these numbers can be stored/searched/ordered as numbers in elasticsearch.

Reply:
yep, grepped osquery repo, all the tables use BIGINT/INTEGER field type, except these two:

```
table_name("process_file_events")
Column("uid", TEXT, "The uid of the process performing the action"),

table_name("disk_encryption")
Column("uid", TEXT, "Currently authenticated user if available"),
```